### PR TITLE
Allow `ignore` command with phpcs

### PR DESCRIPTION
**Pull request by zoidbergwill**

### CODE REVIEW

File: lintreview/tools/phpcs.py [hunk 1]
```diff
@@ -48,6 +48,9 @@ def create_command(self, files):
         standard = 'PSR2'
         if self.options.get('standard'):
             standard = self.apply_base(self.options['standard'])
+        if self.options.get('ignore'):
+            ignore = self.options['ignore']
+            command.append('--ignore=' + ignore)
```
Comment (markstory) [inline lintreview/tools/phpcs.py]: Does this need to be shell escaped? It might be wise to append both `--ignore=` and `ignore` into the array.

Comment (zoidbergwill) [inline lintreview/tools/phpcs.py]: Yeah, I'm re-working this to work a bit more like the Flake8 implementation, it's a lot cleaner.

Comment (markstory) [inline lintreview/tools/phpcs.py]: Cool.

Comment (zoidbergwill) [inline lintreview/tools/phpcs.py]: So phpcs needs to have `--ignore=` 😞

Comment (markstory) [inline lintreview/tools/phpcs.py]: We should probably shell escape then. Paths could contain spaces

Comment (markstory) [inline lintreview/tools/phpcs.py]: Nevermind me, `--ignore=` is for lint rules to ignore not paths. I'm a muppet.

**Pull request merged**